corehq/util/context_processors.py

```diff
@@ -115,6 +115,19 @@ def js_api_keys(request):
     }
 
 
+def js_toggles(request):
+    if not hasattr(request, 'couch_user') or not request.couch_user:
+        return {}
+    if not hasattr(request, 'project'):
+        return {}
+    from corehq import toggles, feature_previews
+    domain = request.project.name
+    return {
+        'TOGGLES_DICT': toggles.toggle_values_by_name(username=request.couch_user.username, domain=domain),
+        'PREVIEWS_DICT': feature_previews.preview_values_by_name(domain=domain)
```

Comment:
nit: what do you think of changing these keys to lower-case? That would make it slightly easier to grep for them to see where they are referenced.

Reply:
sure => 301f5986f64